mscore/startcenter.ui

```diff
@@ -68,7 +68,7 @@
        <item>
         <widget class="QPushButton" name="openScore">
          <property name="text">
-          <string>Open a score…</string>
+          <string>Open a Score…</string>
```

Comment:
Is that even right? The “Open a score…” reads better and as ‘more correct’ to me.

Reply:
General rule of thumb is to use title case for menus and buttons, sentence case elsewhere.  Different style guides recommend differently, but this is most common, and it's what we try to be consistent about.

BTW, one that bugs me is Apply to all Parts, I think "All" should be capitalized as well.